site/en/docs/devtools/recorder/index.md

```diff
@@ -55,20 +59,24 @@ In the next sections, we will walk you through how to record, replay and audit t
 ## Record a user flow {: #record }
 
 1. Open this [demo](https://coffee-cart.netlify.app/) page. Click on the **Start new recording** button to begin.
-2. Enter "coffee checkout" in the **Recording name** textbox, then click on the **Start a new recording** button.
-    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/FLa973eE3tGGhHomHkOc.png", alt="Start a new recording", width="800", height="528" %}
-3. The recording is started. The panel is showing **Recording...** indicating the recording is in progress.
+1. Enter "coffee checkout" in the **Recording name** textbox.
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/EnRcThJpdtY6c91ZTRMT.png", alt="Start a new recording", width="800", height="520" %}
+   Notice the default `data-testid` value in the **Selector attribute** textbox and leave it as it is for the purposes of this tutorial.
+   {% Aside %}
+   If you use a custom selector attribute in your code, this is where you can specify it. For more information, see [Customize the recording's selector](#customize-selector).
```

Comment:
I think some users might it surprise that it detected `data-testid`. instead of `id` or CSS class. 

Explain and reasoning: 
- Suggest to explain in the `aside` (or add one more aside) to explain that `data-testid` is optional, it will work fine with id or class.
- Suggest to explain the reasoning on the `data-testid` usage. 

Can reference to https://docs.cypress.io/guides/references/best-practices#Selecting-Elements and the `Did you know` section.